setup_undercloud.yml

```diff
@@ -94,10 +82,7 @@
 
         - name: set undercloud to PXE boot off Foreman (Dell)
           shell: |
-              source {{ badfish_venv }}/bin/activate
-              python3 badfish.py -H mgmt-{{ undercloud_hostname }} -u quads -p {{ chassis_password }} -i config/idrac_interfaces.yml --boot-to-type foreman --pxe
-          args:
-            chdir: "{{ ansible_user_dir }}/badfish"
+              docker run --rm --dns {{ dns_ip }} quads/badfish -H mgmt-{{ undercloud_hostname }} -u quads -p {{ chassis_password }} --pxe
```

Comment:
Use podman and you don't need DNS
For example,
podman run -it --rm {{ badfish_pod_image }} -u  quads-p {{ chassis_assword }} -i config/idrac_interfaces.yml